hasura/metadata/databases/databases.yaml

```diff
@@ -0,0 +1,12 @@
+- name: default
+  kind: postgres
+  configuration:
+    connection_info:
+      database_url: postgres://eoscr:password@postgres:5432/localdb
```

Comment:
@tetogomez I think that you generated these migrations files using the version 3 for metadata, I think that for this project we should keep using the version 2, so to rollback to the previous version please change the version at `hasura/config.yaml` to `version: 2` or do a rebase with `dev` branch because it already have the proper version and then remove `hasura/metadata/databases/` folder generated again your migration